test/e2e/test-plugins/templates.php

```diff
@@ -0,0 +1,46 @@
+<?php
```

Comment:
If we're going to have a dozen or more test plugins eventually, does it make sense to have a more verbose file naming scheme than `templates.php`?

Reply:
I see us having probably a dozen of plugins here. I'm open to suggestions on the naming. It's not that critical at the moment.

Reply:
Yes, let's refactor when this becomes an issue. It's going to be easier to come up with a pattern when we have more files there.